qendpoint-core/src/main/java/com/the_qa_company/qendpoint/core/hdt/impl/TempHDTImporterOnePass.java

```diff
@@ -53,7 +53,7 @@ public TripleAppender(TempDictionary dict, TempTriples triples, ProgressListener
 		}
 
 		@Override
-		public void processTriple(TripleString triple, long pos) {
+		synchronized public void processTriple(TripleString triple, long pos) {
```

Comment:
I think it would be better to propose a sync and unsync version (or a wrapper) to avoid a sync during a single thread usage